gorm/converter.go

```diff
@@ -0,0 +1,340 @@
+package gorm
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/protoc-gen-go/generator"
+
+	"github.com/infobloxopen/atlas-app-toolkit/query"
+	"github.com/infobloxopen/atlas-app-toolkit/rpc/resource"
+)
+
+type DefaultPbToOrmConverter struct {
+	processor FilteringConditionProcessor
```

Comment:
Shall we use composition there ? Maybe it better to implement all interfaces in separated structs and after, embed them into DefaultPbToOrmConverter like we did in CollectionOperatorsConverter?